lodash.src.js

```diff
@@ -11156,6 +11156,212 @@
+      return minuend - subtrahend;
+    }
+
+    /**
+     * Multiplies a number by another.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} multiplicand The first number to multiply.
+     * @param {number} multiplier The second number to multiply.
+     * @returns {number} Returns the product.
+     * @example
+     *
+     * _.multiply(6, 4);
+     * // => 24
+     */
+    function multiply(multiplicand, multiplier) {
+      return multiplicand * multiplier;
+    }
+
+    /**
+     * Divides a number from another.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} dividend The number to divide.
+     * @param {number} divisor The number to divide by.
+     * @returns {number} Returns the quotient.
+     * @example
+     *
+     * _.divide(6, 4);
+     * // => 1.5
+     */
+    function divide(dividend, divisor) {
+      return dividend / divisor;
+    }
+
+    /**
+     * Calculates the remainder of a number divided by another.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} dividend The number to divide.
+     * @param {number} divisor The number to divide by.
+     * @returns {number} Returns the remainder.
+     * @example
+     *
+     * _.modulus(6, 4);
+     * // => 2
+     */
+    function modulus(dividend, divisor) {
+      return dividend % divisor;
+    }
+
+    /**
+     * Calculates the result of a number raised to the power of another.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} base
+     * @param {number} exponent
+     * @returns {number}
+     * @example
+     *
+     * _.power(6, 4);
+     * // => 1296
+     */
+    function power(base, exponent) {
+      return Math.pow(base, exponent);
+    }
+
+    /**
+     * Calculates the result of a number rounded to an optional precision.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} number The number to round.
+     * @param {number} [precision] The precision to round to.
+     * @returns {number} Returns the rounded number.
+     * @example
+     *
+     * _.round(4.006);
+     * // => 4
+     *
+     * _.round(4.006, 2);
+     * // => 4.01
+     *
+     * _.round(4060, -2);
+     * // => 4100
+     */
+    function round(number, precision) {
+      if (precision) {
+        precision = power(10, precision);
+        return Math.round(number * precision) / precision;
+      } else {
+        return Math.round(number);
+      }
+    }
+
+    /**
+     * Calculates the sum of an array of numbers.
+     *
+     * @static
+     * @memberOf _
+     * @category Math
+     * @param {number} array The numbers to add.
+     * @returns {number} Returns the sum.
+     * @example
+     *
+     * _.sum([4, 6, 2]);
+     * // => 12
+     */
+    function sum(array) {
+      if (!isArray(array) || array.length === 0) return NaN;
+      return reduce(array, add);
```

Comment:
> I thought the idea here wasn't to call other lodash functions directly?

It won't be, but I rather not nit the PR to death and get it in, then tweak after the fact.